src/Examples.Utils/AG_NEWSReader.cs

```diff
@@ -3,41 +3,41 @@
 using System.IO;
 using System.Linq;
 using TorchSharp;
-using TorchSharp.Tensor;
+using static TorchSharp.torch;
 
 namespace TorchText.Data
 {
     public class AG_NEWSReader : IDisposable
     {
-        public static AG_NEWSReader AG_NEWS(string split, Device device, string root = ".data")
+        public static AG_NEWSReader AG_NEWS(string split, device device, string root = ".data")
         {
             var dataPath = Path.Combine(root, $"{split}.csv");
             return new AG_NEWSReader(dataPath, device);
         }
 
-        private AG_NEWSReader(string path, Device device)
+        private AG_NEWSReader(string path, device device)
```

Comment:
Regarding the lower case `device` type - I was expecting type names would stay uppercase?

Reply:
Yes, because Python class names are usually Capitalized. However, this specific type is all lowercase in Pytorch, and I went with the same in TorchSharp, for inter-language consistency.

Reply:
Hmmmm.  I think for this one I'd go uppercase, with a lower-case `device` value with instance members to allow `torch.device.CPU`.   I just can't quite handle seeing `device device` in C# code :-)